pkg/html/html.go

```diff
@@ -7,21 +7,43 @@ import (
 	"fmt"
 	"html/template"
 	"net/url"
+	"path/filepath"
 	"reflect"
 	"strings"
 	texttemplate "text/template"
 	"time"
 
 	"github.com/google/syzkaller/dashboard/dashapi"
 	"github.com/google/syzkaller/pkg/vcs"
+	"google.golang.org/appengine"
 )
 
+// searchPath differs for the tests and AppEngine because we don't follow
+// the recommended folder structure (my best guess).
+// When you run the dashboard/app tests, CWD is syzkaller/dashboard/app.
+// When you deploy AppEngine in the GOPATH mode, CWD is syzkaller/dashboard/app.
+// When you deploy AppEngine in the  GOMOD, CWD is syzkaller/.
+func globSearchPath() string {
+	if appengine.IsAppEngine() {
+		return "dashboard/app/"
```

Comment:
Do we need to also prefix `.txt` templates used in https://github.com/google/syzkaller/blob/master/dashboard/app/reporting_email.go ? And `.html` files used in https://github.com/google/syzkaller/blob/master/dashboard/app/main.go ?

Reply:
They are prefixed here. That lines call this function.